src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -47,6 +52,8 @@ MXNET_OPERATOR_REGISTER_UNARY_LAUNCH_DR(sigmoid, cpu, kernel_launch_op::sigmoid)
 .. math::
    y = 1 / (1 + exp(-x))
 
+The storage type of ``sigmoid`` output is always denseThe storage type of ``sigmoid`` output is always dense
```

Comment:
Is it more consistent to say that the output storage type is `default` instead of `dense`?